migrations/20180512175804_add_hxl_metadata_table.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddHxlMetadataTable extends AbstractMigration
+{
+    public function up()
+    {
+        $this->table('hxl_metadata')
```

Comment:
Code climate complains because my migration is too long. I think in the case of a table definition, separating it would not increase readability, but open to changing it if someone disagrees